example/app/app.component.html

```diff
@@ -3,20 +3,27 @@ <h1>&lt;angular-custom-modal&gt;</h1>
 
   <div id="actions">
     <button class="big" (click)="componentInsideModal.open()">Component inside modal</button>
-    <!-- Alternatively we can define the open call on the component 
+    <!-- Alternatively we can define the open call on the component
       <button class="big" (click)="openFromComponent()"></button> -->
     <button class="big" (click)="htmlInsideModal.open()">Raw HTML inside modal</button>
+    <button class="big" (click)="htmlInsideModalWhitNestedModal.open()">Modal with nested modal</button>
 
     <modal #componentInsideModal>
-      <ng-template #modalHeader><h2>Component inside modal</h2></ng-template>
+      <ng-template #modalHeader>
+        <button class="close" data-dismiss="modal" type="button" aria-label="Close" (click)="componentInsideModal.close()">×</button>
```

Comment:
Try to keep the demo source code as simple as possible. New users who check the library will find it easier to understand. In this case, if the library already provides a close button, this is redundant, and should only be mentioned in the documentation as advanced usage (or maybe as a more advanced demo case too).

Reply:
I also removed it from the lib. I did that, because in one of my use case, I want to control what happens when this button is pressed, also I use other icon. What do you think to have a default implementation, and remove this from the original example, but add a new button that has a personalized close button?

Reply:
corrected. just push my master again?? or make a new PR?